platform/security/idp/security-idp-client/src/main/java/org/codice/ddf/security/idp/client/IdpHandler.java

```diff
@@ -494,7 +494,12 @@ private String createAndSignAuthnRequest(boolean isPost, boolean wantSigned)
         issuer.setValue(spIssuerId);
         authnRequest.setIssuer(issuer);
 
-        authnRequest.setAssertionConsumerServiceURL(spAssertionConsumerServiceUrl);
+        if (idpMetadata.getAssertionConsumerServiceURLCheck()){
```

Comment:
This is fine for 2.10.x, but this is fixed in 2.11.0 where we now have a different host/port for exposing services from within a container and shouldn't be needed when we port this upstream.